draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -3050,6 +3050,62 @@ The constants used to compute y\_den are as follows:
 - k\_(4,1) = 0x1a0111ea397fe69a4b1ba7b6434bacd764774b84f38512bf6730d2a0f6b0f6241eabfffeb153ffffb9feffffffffa9d3 * I
 - k\_(4,2) = 0x12 + 0x1a0111ea397fe69a4b1ba7b6434bacd764774b84f38512bf6730d2a0f6b0f6241eabfffeb153ffffb9feffffffffaa99 * I
 
+# Cofactor Clearing {#clear-cofactor-suites}
+
+Some elliptic curve groups, e.g. G2 in BLS12-381, have a very large cofactor h\_eff (cf. {{cofactor-clearing}}).
+For these curves, we propose an alternative method for clearing the cofactor.
```

Comment:
The existence of faster methods is already stated in `{{cofactor-clearing}}`, so that fact should not be repeated here.

In addition, it does not seem correct to say that "we propose" this method---*Budroni and Pintore* proposed this method, based on `{{SBCDK09}}` and `{{FKR11}}`.

But: I wouldn't say any of that here, since it's redundant with the text in `{{cofactor-clearing}}`. Instead, I'd forward ref from the bullet there to here and then this section should jump right into example code.

Reply:
Fixed, thanks!